### DISCUSSION TITLE
Unit_incremental_heat_rate=0 is not being imported

### DISCUSSION THREAD

**Discussion by danajhen** [Spine-project/Spine-Toolbox#2671] [Support]
Hi,

We are using the unit_idle_heat_rate which requires any unit_incremental_heat_rate which we just set to 0 in our input excel. However, it is not being imported using the standard importer, and we have to add it back manually. Is there any way to import parameters that have 0 as a value? Any suggestions are welcome!

Thanks in advance!

**Comment by soininen** [comment]
Which Excel file are you referring to? FlexTool's input Excel?

**Reply by hannesfelipe** [reply]
We are sorry for the confusion (I work together with @danajhen) We don't use one of the examples. We have our model and import the inputs from a .xlsx file. 

However, I think we made an error when preparing the input rows of the .xlsx. We will check this and I will let you know as soon as we have checked it.
